### DISCUSSION TITLE
[Question] Will Factory and IndustReal environments be available in IsaacLab?

### DISCUSSION THREAD

**Discussion by WuDuidi** [isaac-sim/IsaacLab#1057] [Q&A]
Hello NVIDIA Isaac Team,

I have been transitioning from IsaacGym to IsaacLab as part of my work, and I noticed that two of the environments I frequently use, Factory and IndustReal from IsaacGymEnvs, do not currently seem to be available in IsaacLab. These environments are crucial for my research, and I am keen to continue using them in the IsaacLab framework.

### Questions:

Are there plans to include the [Factory](https://github.com/isaac-sim/IsaacGymEnvs/blob/main/docs/factory.md) and [IndustReal](https://github.com/isaac-sim/IsaacGymEnvs/blob/main/docs/industreal.md) environments from IsaacGymEnvs in future releases of IsaacLab? If so, is there a timeline or roadmap for when these environments might be available?

Are there any alternative environments in IsaacLab that could serve as temporary replacements for Factory and IndustReal, in terms of functionality or learning objectives?

For less experienced users, migrating the Factory and IndustReal environments to IsaacLab manually presents a considerable challenge, and I would greatly appreciate it if the team could consider including these tasks in future IsaacLab updates. Many thanks in advance for considering this request.

**Comment by glvov-bdai** [comment]
You can see some of the planned environments in https://github.com/isaac-sim/IsaacLab/issues/748   


 It looks like the nut and bolt example is in progress.
